stablehlo/tests/CheckOps.cpp

```diff
@@ -156,6 +157,96 @@ llvm::Error evalExpectSerializedEqOp(const Tensor &expected, StringRef probeId,
   return evalExpectEqOp(expected, *tensor);
 }
 
+static uint64_t ULPDifference(APFloat f, APFloat g) {
+  // z is ULP-distance between exact 0 and largest subnormal:
+  uint64_t z = APFloat::getSmallestNormalized(f.getSemantics())
+                   .bitcastToAPInt()
+                   .getLimitedValue() -
```

Comment:
Assert that the float can fit in 64-bits. This is always true in practice since no float is currently more than 64 bits, but this will currently silently fail if an FP128 type is ever introduced.

Reply:
Done.